profile/utils.js

```diff
@@ -332,7 +332,7 @@ async function getSelfUser() {
 }
 
 async function getUser(userId) {
-  const userResponse = await fetch(`${API_BASE_URL}/users/userId/${userId}`, {
+  const userResponse = await fetch(`${API_BASE_URL}/users/?id=abc`, {
```

Comment:
The id need not be hardcoded, it should be passed by the user. @Srinja333

Reply:
means users/${userId}??

Reply:
@vikhyat187 i have updated as per your comment please heck it